psychopy/iohub/client/__init__.py

```diff
@@ -610,7 +611,7 @@ def createTrialHandlerRecordTable(self, trials, cv_order=None):
 
         for cond_name in self._cv_order:
             cond_val = trial[cond_name]
-            if isinstance(cond_val, basestring):
```

Comment:
If we switch this line then unicode strings passed from Py2 will raise errors. I *think* you can leave it as basestring but then do thisi import at top: `from past.builtins import basestring`
see http://python-future.org/compatible_idioms.html